### DISCUSSION TITLE
Colored required fields

### DISCUSSION THREAD

**Discussion by agruenberg** [Budibase/budibase#8085] [Help] [answered]
Is there any other possibility to make all required fields in a form in an other color, than to edit each single CSS?

**Comment by andz-bb** [comment, marked as answer]
hey @agruenberg

I wasn't able to find a way to target required form fields specifically using CSS so you may have to style each component individually unfortunately. If you are going to be repeating the same code a lot then it could be worth making your own form component specifically for this. There are [docs and an example here ](https://docs.budibase.com/docs/custom-component) on how to make custom Budibase form components. This would give you the most control over your field's styling.
